app/views/unattended/report_templates/jobs_-_invocation_report_template.erb

```diff
@@ -30,15 +30,11 @@ require:
 <%- load_hosts(search: search).each do |batch| -%>
 <%-   batch.each do |host|  -%>
 <%-     task = invocation.sub_task_for_host(host) -%>
+<%-     outputs = { 'stdout' => [], 'stderr' => [], 'debug' => [] }  -%>
 <%-     task.action_continuous_output.each do |output| -%>
-<%-       report_row(
-            host: host.name,
-            result: task.result,
-            type: output['output_type'],
-            message: output['output'],
-            time: format_time(output['timestamp']),
-          ) -%>
+<%-       outputs[output['output_type']] << output['output'] -%>
 <%-     end -%>
+<%-     report_row(host: host.name, stdout: outputs['stdout'], stderr: outputs['stderr'], debug: outputs['debug']) -%>
```

Comment:
It would be also good to keep the time information. I'd call it "finished at" and we should put the time when the job finished for a given host.